andela/AndelaEatsUI
# #160447114 Implement add meal modal

#### What does this PR do?
- Adds modal for admin to add meals

#### Description of Task to be completed?
- Include and style for admin add meal modal
- Write add modal component test
- Remove vendor from meal cards

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/meals)
- You should see an `Add meal` button. Click the button
- A modal should popup to add a meal

#### What are the relevant pivotal tracker stories?
- [#160447114](https://www.pivotaltracker.com/story/show/160447114)

#### Background Context
* This is a UI only task, the functionality will be added in a separate story

#### Screenshots
<img width="1440" alt="screen shot 2018-09-13 at 5 49 31 pm" src="https://user-images.githubusercontent.com/32123313/45503023-7d8d3000-b77d-11e8-9a17-ecc463f537e4.png">

<img width="364" alt="screen shot 2018-09-13 at 5 51 39 pm" src="https://user-images.githubusercontent.com/32123313/45503082-b75e3680-b77d-11e8-8179-166874f64365.png">
